YetAnotherGame/Assets/Scripts/Controllers/PlayerController.cs

```diff
@@ -10,16 +10,33 @@ public class PlayerController : MonoBehaviour
 
     private Vector3 moveDirection = Vector3.zero;
     private CharacterController controller;
+    
+    [SerializeField]
+    [Header("First point view camera position")]
+    private Transform fpv = null;
```

Comment:
Может следует дать более осмысленные имена? Например, firstPersonTransform
```suggestion
    private Transform firstPersonViewTransform= null;
```